Configuration/SwiftUITestHost.xctestplan

```diff
@@ -0,0 +1,24 @@
+{
+  "configurations" : [
+    {
+      "id" : "AC613D2C-06BC-4F40-AE71-3AEB7D85C05F",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "mainThreadCheckerEnabled" : false
```

Comment:
We want the main thread checker enabled for this (and UI tests are an example of a case where it might even be useful). It's only disabled in the Xcode 11 version of the test plans, not the default one.